### ISSUE TITLE
Writing and reading back an aggregated table fails on parquet.

### ISSUE THREAD

**Issue by jcferretti** [deephaven/deephaven-core#726]
This new test case for TestTableManagementTools fails:

```
    private Table getAggregatedResultTable() {
        final int size = 40;

        final String [] symbol = new String[size];
        final double [] bid = new double[size];
        final double [] bidSize = new double[size];

        for (int ii = 0; ii < size; ++ii) {
            symbol[ii] =  (ii < 8) ? "ABC" : "XYZ";
            bid[ii] =  (ii < 15) ? 98 : 99;
            bidSize[ii] =  ii;
        }

        final Table baseTable = newTable(stringCol("USym", symbol), doubleCol("Bid", bid), doubleCol("BidSize", bidSize));
        return baseTable.by("USym", "Bid").by("USym");
    }

    @Test
    public void testWriteAggregatedTable() {
        String path = testRoot + File.separator + "testWriteAggregatedTable";
        final Table table = getAggregatedResultTable();
        TableManagementTools.writeTable(table, path);
        Table readBackTable = TableManagementTools.readTable(new File(path));
        TableTools.show(readBackTable);
        TableTools.show(table);
        TestTableTools.tableRangesAreEqual(table, readBackTable, 0, 0, table.size());
        readBackTable.close();
    }
```

**Comment by rcaudy** [comment]
This can be addressed by porting the fix for https://deephaven.atlassian.net/browse/DH-11035 .

**Comment by rcaudy** [comment]
Hmm, well, I'm pretty sure it can, but maybe it requires more of the magic in https://github.com/deephaven/deephaven-core/pull/617 to work; the unit tests are still failing.

**Comment by rcaudy** [comment]
Fixed by #727
